### ISSUE TITLE
Collection name override not working

### ISSUE THREAD

**Issue by Dids** [vadimdemedes/mongorito#185]
Here's my model
```
import { Model } from 'mongorito'

class Commit extends Model {
  collection () {
    return 'my_api_commits'
  }
}

module.exports = Commit
```

It ends up as a collection called `commits`, so it's like it's not even calling the method.

**Comment by mbale** [comment]
What if you try it with static collection?

**Comment by Dids** [comment]
@mbale Could you elaborate on that?

**Comment by vadimdemedes** [comment]
Confirmed the bug, thanks for reporting!

**Comment by vadimdemedes** [comment]
https://github.com/vadimdemedes/mongorito/commit/4977e459a0acabc87fdd155875ed07ddd5fb26e1 should fix this.

**Comment by vadimdemedes** [comment]
Released `3.0.2` with a fix.
